styles/base.css

```diff
@@ -0,0 +1,51 @@
+body {
+  /*position: relative;*/
```

Comment:
Careful here with commented out code. This is what version control (git) is for.